### PR TITLE
Taking down the website.

### PR DESCRIPTION
**Pull request by IvanSanchez**
Implements #8024.

### CODE REVIEW

**Comment by IvanSanchez** [comment]
I was hesitant to add links to charities that support military action. To be honest, I feel that's a moral quagmire right now. I've added that (albeit with a passive language) as the sites linked by @ghybs & @Malvoz in #8024 point to that as well.

The `404.md` file should work with github pages, but cannot be properly tested until deployed. All content is still available, after following a link at the bottom of the message. Changing the file structure seems to not have broken anything AFAICT.

**Comment by Falke-Design** [comment]
I would mark this section as a quote 
> Like this or _cursive_

grafik

**Comment by Falke-Design** [comment]
The start page has a small error:
grafik

**Comment by Falke-Design** [comment]
@IvanSanchez It looks like you didn't used the newest version from the docs before copying it into `SlavaUkraini`. Maybe you can copy the newest docs files again if it is not to much work.

**Review by Falke-Design — APPROVED** [review]
I checked it locally looks good. Thank you!

Maybe you can copy the newest files but this should not block it:
> @IvanSanchez It looks like you didn't used the newest version from the docs before copying it into `SlavaUkraini`. Maybe you can copy the newest docs files again if it is not to much work.

**Comment by IvanSanchez** [comment]
> @IvanSanchez It looks like you didn't used the newest version from the docs before copying it into `SlavaUkraini`. Maybe you can copy the newest docs files again if it is not to much work.

I just did, even from a fresh clone of the repo. My `git diff`s show no changes, and the diff in this PR looks clean - everything shows as renaming without changes.

File: README.md [hunk 1]
```diff
@@ -1,3 +1,28 @@
...
+
+The Russian occupation of Ukraine contravenes and violates several international laws, including the United Nations Charter. It is an unprovoked, unjustifiable, and despicable action that endangers countless civilians.
+
+As Vladimir [expressed a few days ago](https://twitter.com/LeafletJS/status/1496051256409919489):
+
+> If you want to help, educate yourself and others on the Russian threat, follow reputable journalists, demand severe Russian sanctions and Ukrainian support from your leaders, protest war, reach out to Ukrainian friends, donate to Ukrainian charities. Just don't be silent.
+
+Ukrainians are recommending the [https://savelife.in.ua/en/donate/](https://savelife.in.ua/en/donate/) charity. Consider also the [Ukrainian Red Cross](https://redcross.org.ua/en/).
+
+If an appeal to humanity doesn't work for you, I'll appeal to your egoism: the future of Ukraininan citizens is the future of Leaflet.
```
Comment (ghybs) [inline README.md:13]: Minor typo "Ukraininan" should be "Ukrainian"

**Comment by ghybs** [comment]
The website looks fine on mobile after deployment, good job! 👌

File: README.md (hunk 1 above)
Comment (IvanSanchez) [reply README.md:13]: Thanks for noticing! I just force-pushed the typo fix to main: 3bf06774

**Pull request merged**